strawberry/sanic/graphiql.py

```diff
@@ -0,0 +1,13 @@
+from os.path import abspath, dirname, join
+
+
+def render_graphiql_page():
+    dir_path = abspath(join(dirname(__file__), ".."))
+    graphiql_html_file = f"{dir_path}/static/graphiql.html"
+
+    html_string = None
+
+    with open(graphiql_html_file, "r") as f:
+        html_string = f.read()
```

Comment:
Is this .html file supposed to be committed? I'm not personally familiar with the GraphiQL system. At minimum, this may benefit from a bit of comments/docstring?

Also, I would use `Pathlib` instead of `os.path`. This project doesn't need to support old Python versions. [`Path.read_text()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text) is really convenient.

Reply:
The `html` file is actually a part of the strawberry project itself, and was not added in this PR. We are just using the established code practice in `strawberry/flask/graphiql.py` with some minor `sanic` changes.

Reply:
Haha, I see: https://github.com/strawberry-graphql/strawberry/search?q=graphiql.html

There are already 4 different ways were accessing this file. I wonder if we should abstract it out (in another PR).